test/integration/containerized/test_cli_containerized.py

```diff
@@ -6,43 +6,49 @@
 HERE = os.path.abspath(os.path.dirname(__file__))
 
 
+@pytest.fixture
+def skip_if_no_podman(container_runtime_installed):
+    return
+    if container_runtime_installed != 'podman':
+        pytest.skip('podman container runtime(s) not available')
+
+
 @pytest.mark.serial
-def test_module_run(cli, container_runtime_available):
-    if not container_runtime_available:
-        pytest.skip('container runtime(s) not available')
-    cli(['-m', 'ping','--hosts', 'localhost', 'run', os.path.join(HERE, 'priv_data')])
+def test_module_run(cli, skip_if_no_podman):
+    r = cli(['run', '-m', 'ping','--hosts', 'localhost', os.path.join(HERE, 'priv_data')])
+    assert '"ping": "pong"' in r.stdout
 
 
 @pytest.mark.serial
-def test_playbook_run(cli, container_runtime_available):
-    if not container_runtime_available:
-        pytest.skip('container runtime(s) not available')
-    cli(['run', os.path.join(HERE,'priv_data'), '-p', 'test-container.yml'])
+def test_playbook_run(cli, skip_if_no_podman):
+    r = cli(['run', os.path.join(HERE,'priv_data'), '-p', 'test-container.yml'])
+    assert 'PLAY RECAP *******' in r.stdout
+    assert 'failed=0' in r.stdout
```

Comment:
Again, this is the best assertion I've got. As the prior state demonstrates, we need to make _some_ assertion or else the test can pass as a false-pass.